packages/geany_plugins.rb

```diff
@@ -3,33 +3,50 @@
 class Geany_plugins < Package
   description 'Geany plugins.'
   homepage 'https://www.geany.org/'
-  version '1.37'
+  version '1.37-7c54'
   license 'GPL-2'
-  compatibility 'aarch64,armv7l,x86_64'
-  source_url 'https://download.geany.org/geany-plugins/geany-plugins-1.37.tar.bz2'
-  source_sha256 'c98f9b1303f4ab9bed7587e749cd0b5594d9136a1bf8ba110900d46a17fa9cd8'
+  compatibility 'x86_64 aarch64 armv7l'
+  source_url 'https://github.com/geany/geany-plugins.git'
+  git_hashtag '7c540bf347d0f0c6f6ab5123fa8f226b2f9152ee'
 
-  binary_url ({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/geany_plugins/1.37_armv7l/geany_plugins-1.37-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/geany_plugins/1.37_armv7l/geany_plugins-1.37-chromeos-armv7l.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/geany_plugins/1.37_x86_64/geany_plugins-1.37-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/geany_plugins/1.37-7c54_armv7l/geany_plugins-1.37-7c54-chromeos-armv7l.tpxz',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/geany_plugins/1.37-7c54_armv7l/geany_plugins-1.37-7c54-chromeos-armv7l.tpxz',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/geany_plugins/1.37-7c54_x86_64/geany_plugins-1.37-7c54-chromeos-x86_64.tpxz'
   })
-  binary_sha256 ({
-    aarch64: '08e0bf95bf7dbf8accb343c29f574513fbcd27259ec967fc4254fe9a8ec2c726',
-     armv7l: '08e0bf95bf7dbf8accb343c29f574513fbcd27259ec967fc4254fe9a8ec2c726',
-     x86_64: 'b18fe4b41730a581e3c3611887d64e44f0514af7e79070a3b3256c7933e05167',
+  binary_sha256({
+    aarch64: '746ecbc8c162e5407a43f64998b5d60f03555b2335d7c626ffc8a59152b9c394',
+     armv7l: '746ecbc8c162e5407a43f64998b5d60f03555b2335d7c626ffc8a59152b9c394',
+     x86_64: '99640a4c250c27ff315ce2184bdd6b6608f9109bc9df80868f5a0fb586ecc0b2'
   })
 
-  depends_on 'geany'
+  depends_on 'libgit2' => :build
+  depends_on 'valgrind' => :build
+  depends_on 'vte' => :build
+  depends_on 'atk' # R
+  depends_on 'cairo' # R
+  depends_on 'enchant' # R
+  depends_on 'gdk_pixbuf' # R
+  depends_on 'geany' # R
+  depends_on 'glib' # R
+  depends_on 'gtk3' # R
+  depends_on 'harfbuzz' # R
+  depends_on 'libsoup2' # R
+  depends_on 'libxml2' # R
+  depends_on 'pango' # R
+  depends_on 'webkit2gtk_4' # R
+  depends_on 'zlibpkg' # R
```

Comment:
Doesn't `depends_on 'geany'` already cover a lot of these dependencies already with gtk3 and sommelier?  Not sure why we need to include all of these.

Reply:
Ah, yes, this is an excuse to fixup geany dependencies too.